### ISSUE TITLE
SleepNumber Auth not working

### ISSUE THREAD

**Issue by kmcquade** [danpenn/SleepIQ#5]
Seems like SleepNumber changed their API. 

I am using your code in a hacky Jenkins job [code](https://github.com/kmcquade/sleep-number-alarm/blob/master/main.go) setup to raise my bed 2x in the morning (it’s been incredible for my sleep quality and I don’t even set any alarms anymore - so thank you!!). 

I noticed I was sleeping in a lot more over the last few weeks and figured out it’s because the script broke. I looked back at the logs and it had this:

login failed -  Login failed - Error #401: Authentication token of type [class com.bam.presentation.rest.endpoints.sleepnumber.auth.CognitoTokenForShiro] could not be authenticated by any configured realms.  Please ensure that at least one realm can authenticate these tokens.

I wanted to ask if you (or anyone else) is running into this same issue and if you have ideas on how to fix it.

**Comment by kmcquade** [comment]
~Could also be something weird with Jenkins but I’m not sure. I figured if someone else is having this issue I’ll get a better idea of where I should focus on a solution~

Edit: I have confirmed that it is not something weird with Jenkins - it is a change in the API from the Sleep Number side. See error below.


![image](https://user-images.githubusercontent.com/3422255/82903442-f3740380-9f2e-11ea-9d64-b11b28f0d5d0.png)
